package.json

```diff
@@ -40,6 +40,7 @@
     "examples/partytown",
     "examples/subscriptions",
     "examples/third-party-queries-caching",
+    "examples/infinite-scroll",
```

Comment:
can we move this line up to behind `examples/express` so they are in alphabetical order?

The order prevent us from having duplicate entry and minimize merge conflict.